source/blog/2019-05-11-rspec-rails-4-0-has-been-released.md

```diff
@@ -0,0 +1,181 @@
+---
+title: RSpec Rails 4.0 has been released!
+author: Benoit Tigeot, Jon Rowe and Phil Pirozhkov
+---
+
+RSpec Rails 4.0 has been released recently! Given our commitment to
+[semantic versioning](http://semver.org/), this update comes with 
+breaking changes for people that are using Rails below 4.2. If we 
+did introduce any bugs, please let us know, and we'll get 
+a patch release out with a fix ASAP.
+
+RSpec continues to be a community-driven project with contributors
+from all over the world. This release includes 258 commits and 56
+merged pull requests from 27 different contributors!
```

Comment:
Github claims [there are 31](https://github.com/rspec/rspec-rails/compare/v3.9.1...v4.0.0), or even [34 contributors](https://github.com/rspec/rspec-rails/compare/v3.9.1...4-0-maintenance), but doesn't provide a list.

Reply:
This stat is supposed to be automated from the rake task, so if its wrong it needs to be fixed in the task

Reply:
We use a git log command. If I do `git log v3.9.1...v4.0.0 | grep Author | sort | uniq`. But we have duplicated account with different emails. With the duplicated account we have 32.

```sh
$ git log v3.9.1...v4.0.0 | grep Author | sort | sed 's/ <.*//' | uniq | wc -l
27
```